Duplicati/Library/Main/Backend/BackendManager.Handler.cs

```diff
@@ -325,19 +316,30 @@ private async Task Run(IReadChannel<PendingOperationBase> requestChannel)
                     for (int i = activeUploads.Count - 1; i >= 0; i--)
                     {
                         var t = activeUploads[i];
-                        activeUploads.RemoveAt(i);
-                        if (t.IsCompleted && !t.IsCanceled && t.Exception != null)
-                            Logging.Log.WriteWarningMessage(LOGTAG, "BackendManagerDisposeError", t.Exception, "Error in active upload: {0}", t.Exception.Message);
+                        if (t.IsCompleted)
```

Comment:
I know that this wasn't present before, but shouldn't this logic also be applied to the downloads?

Reply:
I have refactored this to a method so it is called on both uploads and downloads.
The reason why it was not really there before, is that it does not hurt anything if the downloads are not completed, but for consistency we now also log if these are hanging.

Reply:
Cool, that was what I had in mind - notifying that things didn't go as expected.